README.md

```diff
@@ -1,76 +1,143 @@
 # redux-responsive
+
 A redux reducer for managing the responsive state of your application
 
-# Why Use a Flux Store for Responsive Behavior?
+
+# Why Use Redux for Responsive Behavior?
+
+redux-responsive **does not require that you use React as your view library**.  However, since that is what we use, the explanation of our motivation mentions React.
 
 There are many solutions for cleanly handling responsive designs in React applications. One common approach is to wrap a component in another component which is responsible for handling the behavior and passing the information down as a prop. While this at first seems good and the "react way", as the behavior gets more complicated, this quickly leads to a lot of boilerplate code in a single component. Also, depending on the implementation, it is possible that many copies of the responsive wrapper would create many different resize handlers.
 
-Using a flux store not only reduces the overall noise in a component, but also guarantees that only a single event listener is waiting for resize.
+Using redux not only reduces the overall noise in a component, but also guarantees that only a single event listener is waiting for resize.
 
-# Creating the Store
 
-All you need to do is wrap the ResponsiveStore in your alt instance's createStore method.
+# Setup
 
-```javascript
-// stores/ResponsiveStore.js
+First, add the reducer somewhere in your reducer tree.  It's just a reducer!  Put it wherever you want.  For example, you could put it in your top level call to `combineReducers`.
 
-// import your singleton alt instance
-import alt from 'my-alt-import'
-// import our factory
-import ResponsiveStore from 'alt-responsive'
+```js
+// reducer.js
 
-// pass the store class to alt's wrapper
-export default alt.createStore(ResponsiveStore)
+import {combineReducers} from 'redux'
+import {responsiveStateReducer} from 'redux-responsive'
+
+export default combineReducers({
+    browser: responsiveStateReducer,
+})
 ```
 
-Now your store is ready to use. The store's default breakpoints match common device sizes and are accessible by the following names which are used to indentify them in your view:
+Second, you must add the resize handlers by passing your store to `addResponsiveHandlers`.
```

Comment:
add to what? What do you think about "Second, create the resize handler that will update your store"?